### PR TITLE
Change "I" to "We" to make it sound like a team project

### PR DESCRIPTION
**Pull request by eoduniyi**
This is an example of PR as suggested by the venerable @GegznaV and @bryanhanson

### CODE REVIEW

File: hyperSpec/vignettes/fileio.Rmd [hunk 1]
```diff
@@ -164,7 +164,7 @@ If you need help finding out how to import your data, please search and eventual
 ```{block, type="note", echo=TRUE}
 **Reproducing the Examples in this Manual**
 
-To run the code examples, you should create a folder named "fileio" in your working directory and copy the contents of directory <https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio>.
+To run the code examples, create a folder named "fileio" in your working directory and copy the contents of the directory <https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio>.
 This online directory contains the required datasets (via `git-lfs`).
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:168]: These lines should be corrected after we get rid of `git-lfs` 

The link https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio will also get broken in the future.

File: hyperSpec/vignettes/fileio.Rmd [hunk 2]
```diff
@@ -1199,10 +1199,10 @@ read.txt.Witec.Graph("fileio/txt.Witec/nofilename (Header).txt", encoding = "lat
 This function reads the spectra files automatically, if they are named properly and extracts additional information of the header file. As for the other Witec functions it is possible to read image data by by selecting `type = "map"`{.r}. Line scans and z-stacks should be read as single spectra.
 
 
-# Writing your own Import Function {#sec:writing-Import}
+ unction {#sec:writing-Import}
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1202]: Fix section name

File: hyperSpec/vignettes/fileio.Rmd [hunk 3]
```diff
@@ -1215,10 +1215,10 @@ more details for an already existing import filter.
 \index{PerkinElmer!Fluorescence}
 \index{PerkinElmer!ASCII long}
 
-The raw spectra of the `flu`{.r} data set (see also the respective vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
+The raw spectra of the `flu`{.r} data set (see also the separate vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1218]: Add the link to the vignette (in the same manner Bryan introduced in `hyperSpec.Rmd`)

File: hyperSpec/vignettes/fileio.Rmd [hunk 4]
```diff
@@ -1259,8 +1259,9 @@ to stop with an error, but it is polite to issue an additional warning.
 writeLines(readLines("read.txt.PerkinElmer.R"))
 ```
 
+imports the spectra.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1262]: Random sentence?

File: hyperSpec/vignettes/fileio.Rmd [hunk 5]
```diff
@@ -1288,14 +1289,14 @@ Thus information like the excitation wavelength is lost.
...
+The function `read.ENVI.Nicolet()`{.r} is an excellent example of a more specific import filter derived from a generic filter for the particular file type.
 Nicolet FT-IR Imaging software saves some non-standard keywords in the header file of the ENVI data.
-These information can be used to reconstruct the $x$ and $y$ axes of the images.
+This information can be used to reconstruct the $x$ and $y$ axes of the images.
 The units of the spectra are saved as well.
 
 Function `read.ENVI.Nicolet()`{.r} thus first adjusts the parameters for `read.ENVI()`{.r}.
 Then `read.ENVI()`{.r} does the main work of importing the file.
-The resulting `hyperSpec`{.r} object is post-processed according to the special header entries.
+The resulting `hyperSpec`{.r} object is post-processed according to the unique header entries.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1300]: I'm not sure if "unique" is appropriate here. I didn't read the context, but if I understand correctly, this means "a certain type of headers that trigger something"

File: hyperSpec/vignettes/fileio.Rmd [hunk 6]
```diff
@@ -1308,7 +1309,7 @@ writeLines(readLines("read.ENVI.Nicolet.R"))
 
 ## Deriving import filters for `spc` files
 
-Please note that future changes inside the `read.spc()`{.r} function are likely to occur.
+Please note that future changes inside the `read.spc()`{.r} function is likely to occur.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1312]: ... future changes ... are ... (not "is")

File: hyperSpec/vignettes/fileio.Rmd [hunk 7]
```diff
@@ -1259,8 +1259,9 @@ to stop with an error, but it is polite to issue an additional warning.
 writeLines(readLines("read.txt.PerkinElmer.R"))
 ```
 
+imports the spectra.
 Note how the labels are set.
-The label with the special name `.wavelength`{.r} corresponds to the wavelength axis, all data columns should have a label with the same name.
+The label with the unique name `.wavelength`{.r} corresponds to the wavelength axis, all data columns should have a label with the same name.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1264]: Word "special" (or its synonym) is more appropriate here as `.wavelength` is for `@wavelength` slot of `hyperSpec` object. Maybe "certain" is more appropriate here. Word "unique" does not reflect the essence of the meaning.

File: hyperSpec/vignettes/fileio.Rmd [hunk 8]
```diff
@@ -1232,22 +1232,22 @@ In order to gain speed, the spectra matrix should be preallocated after the firs
...
 explicitly.
--  The same applies for the axis labels: they should default to reasonable settings for
-fluorescence spectra, but it should be possible to change them if needed.
+-  The same applies to the axis labels: they should default to reasonable settings for
+fluorescence spectra, but it should be possible to change them.
 -  The usual log entry arguments should be supplied.
 -  A sanity check should be implemented: stop with an error if a file does not have the same
 wavelength axis as the others.
--  If no file can be found, an empty `hyperSpec`{.r} object is a reasonable result: There is no need
+-  If no file can be found, an empty `hyperSpec`{.r} object is a consistent result: There is no need
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1250]: Shouldn't "There" start with lower-case letter?

File: hyperSpec/vignettes/fileio.Rmd [hunk 9]
```diff
@@ -1215,10 +1215,10 @@ more details for an already existing import filter.
 \index{PerkinElmer!Fluorescence}
 \index{PerkinElmer!ASCII long}
 
-The raw spectra of the `flu`{.r} data set (see also the respective vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
+The raw spectra of the `flu`{.r} data set (see also the separate vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
 
 We need a function that automatically reads all files specified by a pattern, such as `*.txt`{.r}.
-In order to gain speed, the spectra matrix should be preallocated after the first file is read.
+To gain speed, **hyperSpec** preallocates the spectra matrix after the first reading of the file.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:1221]: The meaning of corrected sentence is not the same. Is the new meaning really correct?

File: hyperSpec/vignettes/fileio.Rmd (hunk 1 above)
Comment (eoduniyi) [reply hyperSpec/vignettes/fileio.Rmd:168]: Wait, we still use `git-lfs`?

File: hyperSpec/vignettes/fileio.Rmd [hunk 10]
```diff
@@ -243,9 +243,9 @@ With the arguments:
 ---------------- ---------------------------------------------------------------------------
 
 
-Thus, once your data is in R's workspace, creating a `hyperSpec`{.r} object is easy.
-I suggest wrapping the code to import your data and the line joining it into a `hyperSpec`{.r} object by your own import function.
-You are more than welcome to contribute such import code to package **hyperSpec**.
+hus, once the data is in R's workspace, creating a `hyperSpec`{.r} object is easy.
```
Comment (GegznaV) [inline hyperSpec/vignettes/fileio.Rmd:246]: "hus" ??? Fix into "Thus"

**Review by GegznaV — CHANGES_REQUESTED** [review]
In addition to my remarks, I would like that "one sentence per row" rule would be applied (I see that in some places several sentences per row are used). It makes easier to track and review the text in version control system.

File: hyperSpec/vignettes/fileio.Rmd (hunk 8 above)
Comment (eoduniyi) [reply hyperSpec/vignettes/fileio.Rmd]: https://writingcenter.unc.edu/tips-and-tools/semi-colons-colons-and-dashes/#:~:text=Here's%20our%20suggestion%3A%20generally%2C%20the,to%20capitalize%20it%20or%20not.

File: hyperSpec/vignettes/fileio.Rmd (hunk 1 above)
Comment (GegznaV) [reply hyperSpec/vignettes/fileio.Rmd:168]: we do

File: hyperSpec/vignettes/fileio.Rmd (hunk 4 above)
Comment (GegznaV) [reply hyperSpec/vignettes/fileio.Rmd:1255]: image

It was not obvious in git diff.

**Review by GegznaV — CHANGES_REQUESTED** [review]
One more remark: the branch should have been named as `feature/vign-edits` and not `vign-edits`.

**Review by GegznaV — APPROVED** [review]

**Comment by GegznaV** [comment]
It's enough changes for a single PR. Merging it.

File: hyperSpec/vignettes/fileio.Rmd (hunk 1 above)
Comment (eoduniyi) [reply hyperSpec/vignettes/fileio.Rmd:168]: Oh right, because of large data sets. We got rid of `make`

**Pull request merged**